python/looker_sdk/rtl/api_settings.py

```diff
@@ -49,7 +48,13 @@ def read_config(self) -> Dict[str, str]:
 class ApiSettings(PApiSettings):
     deprecated_settings: Set[str] = {"api_version", "embed_secret", "user_id"}
 
-    def __init__(self, filename: str = _DEFAULT_INI, section: Optional[str] = None):
+    def __init__(
+        self,
+        sdk_version: str,
```

Comment:
Perhaps `sdk_version` should be optional. If the RTL is being used on its own (without the SDK), it might also make sense to rename it, maybe `agent_postfix` would be a more appropriate name. Thoughts?

Reply:
in Typescript, ApiMethods sets the AgentTag in its entirety and pushes that as a property down to the request processor. Maybe the same kind of thing can be used here?